tests/integration_tests/test_contrib_wandb.py

```diff
@@ -15,6 +15,8 @@
 
 @pytest.mark.distributed
 def test_contrib_experiment(csv_filename):
+    os.environ["WANDB_DIR"] = os.path.join(os.getcwd(), 'results')
```

Comment:
Should use a tmpdir for this:

```
def test_contrib_experiment(csv_filename, tmpdir):
    wandb_dir = os.path.join(os.getcwd(), 'results')
    os.makedirs(wandb_dir)
    subprocess.call(['chmod', '-R', '+w', wandb_dir])
    os.environ["WANDB_DIR"] = wandb_dir
```

Reply:
Updated to use tmp

Reply:
Hey @skanjila, the updated code has the same issue. Please see the updated reference code in my comment above (there was a typo before). The important thing to note is use of the `tmpdir` fixture.

Reply:
Whoops sorry @tgaddair I changed the code to use tmpdir, let me know if anything else is missing